extensions/wikia/CommunityPage/templates/allAdmins.mustache

```diff
@@ -1,7 +1,7 @@
 <h2>{{allMembersHeaderText}}</h2>
 <ul class="reset top-contributors">
 	{{#admins}}
-		<li>
+		<li class="contributor">
```

Comment:
Isn't the class name too generic? Maybe we can include a prefix to distinguish Community Page related classes?